### PR TITLE
Genesis change fix

### PR DESCRIPTION
**Pull request by temaniarpit27**

### CODE REVIEW

**Review by vcastellm — APPROVED** [review]
LGTM

File: params/config.go [hunk 1]
```diff
@@ -415,7 +421,8 @@ type BorConfig struct {
 	ValidatorContract     string `json:"validatorContract"`     // Validator set contract
 	StateReceiverContract string `json:"stateReceiverContract"` // State receiver contract
 
-	OverrideStateSyncRecords map[string]int `json:"overrideStateSyncRecords"` // override state records count
+	OverrideStateSyncRecords map[string]int         `json:"overrideStateSyncRecords"` // override state records count
+	BlockAlloc               map[string]interface{} `json:"block_alloc"`
```
Comment (jdkanani) [inline params/config.go:425]:
```suggestion
	BlockAlloc               map[string]interface{} `json:"blockAlloc"`
```

File: params/config.go [hunk 2]
```diff
@@ -281,6 +284,9 @@ var (
 				"14953792": 0,
 				"14953856": 0,
 			},
+			BlockAlloc: map[string]interface{}{
+				"0": "",
```
Comment (jdkanani) [inline params/config.go:288]: Shouldn't this line be the object?

File: params/config.go [hunk 3]
```diff
@@ -245,6 +245,9 @@ var (
 			BackupMultiplier:      2,
 			ValidatorContract:     "0x0000000000000000000000000000000000001000",
 			StateReceiverContract: "0x0000000000000000000000000000000000001001",
+			BlockAlloc: map[string]interface{}{
+				"0": "",
```
Comment (jdkanani) [inline params/config.go:249]: Shouldn't this line be the object?

**Review by jdkanani — CHANGES_REQUESTED** [review]
@ferranbt

**Review by jdkanani — APPROVED** [review]

**Review by ssandeep — APPROVED** [review]

**Comment by miohtama** [comment]
This change looks pretty important, but does not have much metadata and discussion attached to it. What is this change about?

**Pull request merged**